app/assets/javascripts/components/autocomplete.js

```diff
@@ -84,5 +86,53 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     $select.id = $select.id + '-select'
   }
 
+  Autocomplete.prototype.initAutoCompleteWithoutNarrowingResults = function () {
+    // Read options and associated data attributes and feed that as results for inputValueTemplate
+    var $select = this.$module.querySelector('select')
+
+    if (!$select) {
+      return
+    }
+
+    var $options = $select.querySelectorAll('option')
+    var $selectedOption = $select.querySelector('option[selected]')
+    if (!$selectedOption) {
+      $selectedOption = $options[0]
+    }
+    var defaultValue = $selectedOption.textContent
```

Comment:
As we're hoping they can type in their own input should this be dealing with an input field rather than a select box? 

I realise this complicates matters slightly as the option is just "without narrowing results" which doesn't make any implications about input or select.

Reply:
It will be quite tricky since we won't have a way to get the values to populate the dropdown and the autocomplete generates its own input.

Reply:
Sure, might be worth checking with @soniaturcotte then. As far as I was aware, I thought it was essential that a user could type in their own time outside the options?

Reply:
Right, in the current format the users won't be able to enter a custom time if they don't have JavaScript enabled.

Reply:
I don't think they can even if they have JavaScript as the select box is what is submitted as the value rather than the input.

Reply:
In this case the input is what's submitted, not the select. The problem we were seeing was not playing back the user their custom input (as we only got the default items in the list). I've extended the code (with my rusty ruby skills) to include the custom input within the list. Let me know if this makes sense or there's a better approach to it.